packages/connect/src/api/cardano/api/cardanoSignTransaction.ts

```diff
@@ -293,6 +297,13 @@ export default class CardanoSignTransaction extends AbstractMethod<
             if (certificate.key_hash) {
                 this._ensureFeatureIsSupported('KeyHashStakeCredential');
             }
+            if (
+                certificate.type === PROTO.CardanoCertificateType.STAKE_REGISTRATION_CONWAY ||
+                certificate.type === PROTO.CardanoCertificateType.STAKE_DEREGISTRATION_CONWAY ||
+                certificate.type === PROTO.CardanoCertificateType.VOTE_DELEGATION
+            ) {
+                this._ensureFeatureIsSupported('Conway');
```

Comment:
I'd consider making an enum for the strings naming the features.

Reply:
I think that the strings do work like enum values thanks to TypeScript.
https://github.com/vacuumlabs/trezor-suite/blob/d3102fdf0d616aa9b88aad71a65ca59342f399c2/packages/connect/src/api/cardanoSignTransaction.ts#L266